modules/png-extraction/config.json

```diff
@@ -1,13 +1,13 @@
 {
-	"DICOMHome": "/Users/pradeeban/Downloads",
-	"OutputDirectory": "/Users/pradeeban/Downloads/root",
-	"Depth": 0,
-	"SplitIntoChunks": 1,
-	"PrintImages": true,
-	"CommonHeadersOnly": false,
-	"UseProcesses": 0,
-	"FlattenedToLevel": "patient",
-	"is16Bit":true,
-	"SendEmail": true,
-	"YourEmail": "test@test.test"
+    "DICOMHome": "/Users/pradeeban/Downloads",
+    "OutputDirectory": "/Users/pradeeban/Downloads/root",
+    "Depth": 0,
+    "SplitIntoChunks": 1,
+    "PrintImages": true,
```

Comment:
Please remove this file from the commit.

and also the ".pyc" file.

Before you do a git commit, check the status to ensure these two files are not committed.